src/components/panel/panel.js

```diff
@@ -1194,6 +1232,22 @@ MdPanelRef.prototype._addStyles = function() {
 
 
 /**
+ * Updates the position configuration of a panel
+ * @param {MdPanelPosition} position
+ */
+MdPanelRef.prototype.updatePosition = function(position) {
+  if (!this._panelContainer) {
+    throw new Error('Panel does not exist yet. Call open() or attach().');
+  }
+
+  var self = this;
```

Comment:
I think the self variable here is no longer necessary and you can use `this` for both lines below.